### ISSUE TITLE
How can I fix "On entry to DLASCLS parameter number  4 had an illegal value"

### ISSUE THREAD

**Issue by phamkhactu** [OpenStitching/stitching#158]
I have [images](https://drive.google.com/drive/folders/1ud84Ijon4lqho-w3QI5wOwthmqZWLbZN?usp=sharing). Then I config for stitch image. 
```
stitcher = Stitcher(detector="sift", 
                    confidence_threshold=0.0001,
                    range_width=1,nfeatures=1000,
                    crop=False,
                    wave_correct_kind="no")

panorama = stitcher.stitch(images) 

```

I get error: 
```
On entry to DLASCLS parameter number  4 had an illegal value
```
Maybe it comes from a litter point matching. I not sure that.

Do you give me some advance?

**Comment by lukasalexanderweber** [comment]
Can you run `stitcher.stitch_verbose(images)` and analyse the intermediate results?

**Comment by phamkhactu** [comment]
Hi @lukasalexanderweber,

After researching, I have another problem and question:
1.  with config ```crop``` is True or False, it cause a lot off affect in image:
![test](https://github.com/OpenStitching/stitching/assets/42369268/8ec6e3e9-5405-4601-b529-3f52101c3ae6)
![bottom](https://github.com/OpenStitching/stitching/assets/42369268/946eae7e-9caa-4076-b741-1def0f386204)


3. After that, I have 2 image have order, I want to merge. If I keep orig config,  perhaps stitch merge them by horizontal.  How to stitch it follow vertical.
![c1](https://github.com/OpenStitching/stitching/assets/42369268/0840de59-504c-4e86-a759-ebe0b247997b)
![bottom](https://github.com/OpenStitching/stitching/assets/42369268/60b7db97-2feb-4555-824f-26f3502669a7)


Here is my configs:

```
stitcher = Stitcher( 
                    detector="sift", 
                    confidence_threshold=0.0001,
                    nfeatures=1000,
                    range_width=1,
                    crop=False
                    )

```

**Comment by lukasalexanderweber** [comment]
Is 1 the upper a result without cropping and the lower with cropping? I don't understand the second part

**Comment by phamkhactu** [comment]
yes upper as you understand, the second part, I want to combine 2 image for vertical, I hope result stitch as below:
![09_result](https://github.com/OpenStitching/stitching/assets/42369268/1596582f-840f-440e-906c-96d7cae8980d)

**Comment by lukasalexanderweber** [comment]
Do you get an Error? Have you tried wave_correct_kind="vert"?

**Comment by lukasalexanderweber** [comment]
I still don't completely understand your problem
